tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "checkJs": true,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "exactOptionalPropertyTypes": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["es2020"],
+    "module": "node16",
+    "newLine": "lf",
+    "resolveJsonModule": true,
```

Comment:
Do we need this for the way we import `package.json`? If not, I suggest to remove it.

```suggestion
```

Reply:
It is needed

removing it results in:

```
packages/redot-cli/cli.js:9:22 - error TS2732: Cannot find module 'redot/package.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.

9 const proc = require('redot/package.json')
                       ~~~~~~~~~~~~~~~~~~~~

packages/redot-cli/cli.js:10:21 - error TS2732: Cannot find module './package.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.

10 const cli = require('./package.json')
                       ~~~~~~~~~~~~~~~~
```